packit_service/models.py

```diff
@@ -761,3 +771,48 @@ def get_by_pipeline_id(cls, pipeline_id: str) -> Optional["TFTTestRunModel"]:
                 .filter_by(pipeline_id=pipeline_id)
                 .first()
             )
+
+
+class InstallationModel(Base):
+    __tablename__ = "github_installations"
+    installation_id = Column(Integer, primary_key=True)
```

Comment:
please don't make this a PK since we don't control the value - it would also be nice if our IDs started from 1 and not getting random 6 digits numbers